internal/servicetypes/solr.go

```diff
@@ -65,11 +66,14 @@ var solr = ServiceType{
 			Resources: corev1.ResourceRequirements{
 				Requests: corev1.ResourceList{
 					corev1.ResourceCPU:    resource.MustParse("10m"),
-					corev1.ResourceMemory: resource.MustParse("100M"),
+					corev1.ResourceMemory: resource.MustParse("10Mi"),
```

Comment:
Is this an intended change? Seems like a regression.

Reply:
The previous value was `10Mi`
```
legacy/helmcharts/solr/values.yaml:    memory: 10Mi
```